nginx/datadog_checks/nginx/nginx.py

```diff
@@ -232,7 +232,11 @@ def _get_plus_api_data(self, instance, api_url, ssl_validation, auth, plus_api_v
             r = self._perform_request(instance, url, ssl_validation, auth)
             payload = self._nest_payload(nest, r.json())
         except Exception as e:
-            self.log.exception("Error querying %s metrics at %s: %s", endpoint, url, e)
+            if endpoint == "stream/server_zones" or endpoint == "stream/upstreams":
```

Comment:
is there any way to make this more future proof? Like checking if the endpoint starts with `stream/` or putting those endpoints in a list as constants, etc... The concern is we probe a new endpoint in the future and this condition gets bypassed causing a regression.

Reply:
That makes sense, I've put the stream endpoints in their own collection so now they only need to be updated in one spot.